plugins/opusCodec/src/OpusEncoder.cpp

```diff
@@ -0,0 +1,269 @@
+
+#include <PerfStat.h>
+#include <QtCore/QLoggingCategory>
+
+#include "OpusEncoder.h"
+#include "opus/opus.h"
+
+static QLoggingCategory encoder("AthenaOpusEncoder");
+
+static QString error_to_string(int error) {
+    switch (error) {
+    case OPUS_OK:
```

Comment:
Please ident the case 4 spaces from the switch ident see Coding standard 4.1.9 https://github.com/kasenvr/project-athena/blob/kasen/core/CODING_STANDARD.md#419-switchcase-statements